tools/subgraph-example/README.md

```diff
@@ -10,29 +12,35 @@ GravatarRegistry -> GravatarCreated and GravatarUpdated
 
 For more information on subgraphs, check the official TheGraph documentation https://thegraph.com/docs/en/
 
-## Prerequisites:
+## Introduction:
+
+1. [What is TheGraph](https://thegraph.com/docs/en/about)
+2. [How to create a subgraph](https://thegraph.com/docs/en/developing/creating-a-subgraph/)
+3. [The AssemblyScript API](https://thegraph.com/docs/en/developing/assemblyscript-api/)
+
+## Try it out:
+
+### Prerequisites:
 
 Note: Currently this example needs to be executed against older relay and mirror-node versions, until all fixes have been released.
```

Comment:
can we specify which versions work?

Reply:
Updated the readme and added a link to the full custom local-node config

Reply:
Nice!  Thank you